doc/rvv-intrinsic-spec.adoc

```diff
@@ -96,29 +98,33 @@ enum __RISCV_VXRM {
 };
 ```
 
-NOTE: Computations of `vsadd`, `vsaddu`, `vssub`, and `vssubu` do not need the rounding mode, therefore the intrinsics of these instructions do not have the `vxrm` parameter.
+NOTE: Rounding mode does not affect the computations of `vsadd`, `vsaddu`, `vssub`, and `vssubu`, therefore the intrinsics of these instructions do not have the `vxrm` parameter.
 
 NOTE: The RISC-V psABI ^9^ states that `vxrm` is not preserved across calls. Optimization for reducing the number of redundant writes to `vxrm` is a compiler and system specific issue.
 
-[[chapter-control-to-frm]]
+NOTE: Control to the vector fixed-point saturation flag (`vxsat`) ^22^ is not yet covered in the RVV C intrinsics v1.0. We plan to support them in follow-up versions in a compatible way with existing intrinsics in v1.0.
+
+[[control-of-frm]]
 === Control of `frm`
 
-For the floating-point intrinsics, representing the instructions under v-spec section 13, the intrinsics have two classes, called implicit-`frm` and explicit-`frm`.
+For the floating-point intrinsics, representing the floating-point arithmetic instructions ^23^, the intrinsics have two variants, called the implicit-`frm` and the explicit-`frm` intrinsics.
+
+NOTE: Control to the floating-point accrued exceptions flag fields (`fflag`) ^10^ is not yet covered in the RVV C intrinsics v1.0. We plan to support them in follow-up versions in a compatible way with existing intrinsics in v1.0.
 
 ==== Implicit-`frm` intrinsics
 
 The implicit-`frm` intrinsics behave like any C-language floating-point expressions, using the default rounding mode when `FENV_ACCESS` is off, and using the `fenv` dynamic rounding mode when `FENV_ACCESS` is on.
 
 NOTE: Both GNU and LLVM compilers generate scalar floating-point instructions using dynamic rounding mode, relying on the kernel initialization to set `frm` to `RNE` (specified as "roundTiesToEven" in IEEE-754 (a.k.a. IEC 60559)).
```

Comment:
I assume this "kernel" is the OS kernel but maybe this can be expressed more generically as the "environment initialization" because this might be actually done by the loader of the C library.